### DISCUSSION TITLE
G-Code variables

### DISCUSSION THREAD

**Discussion by cochcoder** [OrcaSlicer/OrcaSlicer#5498] [Dev]
Hi, I'm trying to bring support for the Prusa XL with one head and I was wondering if Orca Slicer can transmit the minimum and maximum X and Y print area? This is must likely needed for the Prusa XL's bed as it only heats the needed part. The G-Code in question (from Prusa's slicer) is `M555 X{first_layer_print_min[0]} Y{first_layer_print_min[1]} W{(first_layer_print_max[0]) - (first_layer_print_min[0])} H{(first_layer_print_max[1]) - (first_layer_print_min[1])}`

**Comment by foreachthing** [comment]
Yes, it can.

Open your printer settings, machine g-code, edit custom g-code, Dimensions:

<img width="1178" alt="image" src="https://github.com/SoftFever/OrcaSlicer/assets/10420187/a8c6672f-6642-4db7-a1fa-f66c939b3706">
